sample-project/app/controllers/users_controller.rb

```diff
@@ -54,7 +54,7 @@ def update
   # DELETE /users/1
   # DELETE /users/1.json
   def destroy
-    @user.destroy
+    @user.destroy #test
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vividmuimui/danger-sample/blob/52cb3991be8415173092296f7c5dd1efec42d2b1/sample-project/app/controllers/users_controller.rb#L57"></span>Rails/SaveBang: Use <code>destroy!</code> instead of <code>destroy</code> if the return value is not checked.</td>
    </tr>
  </tbody>
</table>